adobe/aem-core-cif-components
# CIF-1783 - Implement ComponentExporter for product carousel component

## Description

Add the required annotations and getter method to the ProductCarousel Sling Model

## Related Issue

CIF-1783

## Motivation and Context

Expose the Product Carousel configuration via the Sling Model JSON exporter.

## How Has This Been Tested?

Functional testing, unit testing

Unit tests and functional testing

Steps for functional testing:

1/ Install AEM Venia and this branch of the CIF Core Components project
2/ Open the home page in the editor and add a Product Carousel component,
3/ Configure the component by adding some products

4/ Open the model JSON using the model selector, i.e. http://localhost:4502/content/venia/us/en.model.json
5/ Look for the teaser component and check the following:

- The `productIdentifiers` JSON snippet should be an array
- Every identifier should have a `commerceIdentifier` with `entityType=PRODUCT` and `type=SKU`, similar to
 
```json
{
    "id": "-item-4bace57234",
    "commerceIdentifier": {
        "entityType": "PRODUCT",
        "value": "WP07",
        "type": "SKU"
     }
}
```

## Screenshots (if appropriate):

## Types of changes




## Checklist:
